scripts/components/Main.scss

```diff
@@ -30,6 +30,10 @@
   z-index: 4;
 }
 
+.h5p-editor-three-image-wrapper .h5p-three-image.scene-container {
+  overflow: hidden;
```

Comment:
`overflow` is one of those properties where it's often hard to understand why it's used. Would it make sense to add an extra comment here explaining why we use it?